ui/components/multichain/app-header/app-header.js

```diff
@@ -119,7 +128,14 @@ export const AppHeader = ({ location }) => {
 
   return (
     <>
-      {isUnlocked && !popupStatus ? <MultichainMetaFoxLogo /> : null}
+      {isUnlocked &&
+      !popupStatus &&
+      ///: BEGIN:ONLY_INCLUDE_IF(build-experimental)
+      !isSidepanel &&
+      ///: END:ONLY_INCLUDE_IF
+      true ? (
```

Comment:
Is the `true` here just because of the way this is formatted? Is it possible to have the codefence around `&& !isSidepanel` instead?